samples/msal-browser-samples/MsrCryptoSample/package.json

```diff
@@ -8,7 +8,7 @@
   "dependencies": {
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
-    "express": "~4.16.1",
+    "express": "~4.17.3",
```

Comment:
Does this have to be ~ version? Can we make it ^ so we don't have to make this kind of change again in the future?